### PR TITLE
Translated README.md to Hebrew

### LINKED ISSUE #470: Translate the README.md to Hebrew
Hi, 
I'd like to get in to contributing to open source. 
Is there any interest in translating the readme guide to Hebrew?
I'm a native English and Hebrew speaker and would gladly help.

### PR DESCRIPTION
**Pull request by Immanuelbh**
Resolves #470 

* init file

* added rtl

* updated titles

* first section

* completed translation - initial

* fixes

Co-authored-by: Immanuel Ben Hefer <immanuel.ben.hefer@sap.com>

### CODE REVIEW

File: README-HE.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+- [שיטות עבודה מומלצות בגיט](https://sethrobertson.github.io/GitBestPractices/) - בצעו commit לעיתים קרובות, שפרו בהמשך, פרסמו פעם אחת: שיטות העבודה המומלצות בגיט.
+- [מדריך גיט אינטרקטיבי](https://learngitbranching.js.org/) - למדו גיט בדרך הויז׳יאלית והאינטרקטיבית ביותר.
+
+</p>
+</div>
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+
+## ספרים בנושא קוד פתוח
+- [יצירת תוכנה בקוד פתוח](https://producingoss.com/) - זהו ספר על הצד האנושי של פיתוח קוד פתוח. הספר מתאר כיצד פרויקטים מוצלחים עובדים, ציפיות האנשים והמפתחים ותרבות תכונה חינמית.
```
Comment (SoloNoah) [inline README-HE.md:162]: Believe that it should be `תוכנה` and not `תכונה`

File: README-HE.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+[![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+[![Build Status](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
+
+להלן רשימת מקורות עבור מתכנתים מתחילים המעוניינים לתרום לקוד פתוח
+
+אם תמצאו מקורות נוספים, אנא תרמו על ידי יצירת pull request
+
+אם יש לכם שאלות או תגובות, אנא צרו issue חדש
```
Comment (SoloNoah) [inline README-HE.md:40]: I think that in this instance `comments` should be translated to  `הערות`

File: README-HE.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+</p>
+</div>
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+
+## יוזמות לתרומת קוד פתוח
+- [Up For Grabs](https://up-for-grabs.net/) - מכיל פרויקטים עם issues המתאימים למתחילים.
+- [First Timers Only](https://www.firsttimersonly.com/) - רשימה של באגים המתוייגים כ-״first-timers-only".
+- [First Contributions](https://firstcontributions.github.io/) - בצעו את התרומה הראשונה שלכם לקוד פתוח ב-5 דקות. כלי ומדריך שיעזור למתחילים לתרום. תוכלו למצוא את קוד המקור של האתר [כאן](https://github.com/firstcontributions/first-contributions), נצלו את ההזדמנות לתרום ל-repository עצמו.
```
Comment (SoloNoah) [inline README-HE.md:177]: I think that maybe we should change `ב-5 דקות`  to `תוך 5 דקות`
To make it appear more official

**Review by SoloNoah — CHANGES_REQUESTED** [review]
Overall a great job :) Just some small details that I think should be changed.

File: README-HE.md (hunk 1 above)
Comment (Immanuelbh) [reply README-HE.md]: fixed!

File: README-HE.md (hunk 2 above)
Comment (Immanuelbh) [reply README-HE.md]: I think you're right, fixed!

File: README-HE.md (hunk 3 above)
Comment (Immanuelbh) [reply README-HE.md]: It does sound better, fixed!

**Comment by Immanuelbh** [comment]
@erictleung I fixed the typos and the broken link, not sure why the validation fails.

**Review by erictleung — APPROVED** [review]
LGTM! 👍 Thanks for taking time to translate our README to Hebrew!! ✨

**Comment by erictleung** [comment]
@SoloNoah thanks for taking time to review this! Next time, take a look at how to do [suggested changes](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change) in pull requests so that your suggestions can be easily incorporated and you can get credit

**Comment by SoloNoah** [comment]
oops my bad :) kinda new at open sourcing.
Will pay attention next time! thanks for the advice :)

**Pull request merged**